decidim-proposals/lib/decidim/proposals/component.rb

```diff
@@ -265,7 +265,17 @@
         visibility: "all"
       ) do
         proposal = Decidim::Proposals::Proposal.new(params)
-        proposal.add_coauthor(participatory_space.organization)
+        coauthor = case n
+                   when 0
+                     Decidim::User.where(decidim_organization_id: participatory_space.decidim_organization_id).all.sample
+                   when 1
+                     Decidim::UserGroup.where(decidim_organization_id: participatory_space.decidim_organization_id).all.sample
+                   # when 2
+                   # Decidim::Meetings::Meeting.where(decidim_organization_id: participatory_space.decidim_organization_id).all.sample
```

Comment:
```suggestion
                   when 2
                     Decidim::Meetings::Meeting.where(component: component).order(Arel.sql('RANDOM()')).first
```
This should obtain a meeting belonging to the component in the context